### ISSUE TITLE
WEDGE Switch installation

### ISSUE THREAD

**Issue by Ilgar2703** [p4lang/p4c#4988]
I wonder if this is for WEDGE switch installation? In other words, I have an Intel Tofino switch and I need to install it from scratch and run my P4 program developed in the software switch. How do I use this repo for that purpose and do I need any other software packets that I need to have a private access?

Thank you!

**Comment by fruffy** [comment]
This is the open-source compiler, it's tooling is largely independent from Tofino. 

Are you talking about the newly contributed Tofino back end (#4964)? There are still some parts of the full Tofino suite missing but maybe @hanw can clarify.

**Comment by Ilgar2703** [comment]
Yes, I am talking about the new back end. I would appreciate any clarification and help about how to use it for the Tofino switch installation from scratch @hanw . Thanks!

**Comment by Ilgar2703** [comment]
Sorry for asking again, I wanted to learn the situation with the backend since I haven't seen any posts recently. 

Also, since I am in a hurry for a project, is applying this form for the existing setup kit better than waiting for the new backend? https://formfacade.com/public/116697843151790595985/all/form/1FAIpQLSe5JlCTxA2X7EDwK1iYdoFj8bTJk6IjqaHarDHFKhIa482C3w 

Thank you!

**Comment by jafingerhut** [comment]
If it doesn't cost you much time to fill out the form, I'd say go for it.  You have nothing to lose except the time in filling it out (even if there is never a response to you filling out the form -- which I have no idea whether you will get a response before the open source Tofino tools are ready).

**Comment by alexandergall** [comment]
Apart from the Tofino p4c backend, you also need the full Tofino SDE (which contains the p4c Tofino backend) and the reference BSP. Those haven't been open-sourced at all at this point, as far as I know. So currently there is still no way around joining the Intel program, or what's left of it.
